packages/ckb-sdk-rpc/misc/basedirs/lib-esm/package.json

```
@@ -0,0 +1,4 @@
+
+{
+  "type": "module"
+}
```

```suggestion
{
  "type": "module"
}

```
a newline is required at the end while the leading line is unnecessary